New `otelcol.exporter.debug` component (#1043)
grafana/alloy: docs/sources/reference/components/otelcol/otelcol.exporter.logging.md

# otelcol.exporter.logging

{{< admonition type="warning" >}}
The OpenTelemetry Collector `logging` exporter is deprecated.

> It is due to be removed from the upstream Collector repository in September 2024.

Reviewer: ```suggestion
It will be removed from the upstream Collector repository in September 2024.
```

`otelcol.exporter.logging` may also be removed in a future Alloy version.

> Please use `otelcol.exporter.debug` instead.

Reviewer: ```suggestion
Use `otelcol.exporter.debug` instead.
```
